analyzers/NuGet.Config

```diff
@@ -1,9 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
     <clear />
     <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
   </packageSources>
+  <packageSourceCredentials>
+    <Repox_x0020_Artifactory_x0020__x0028_proxy_x0029_>
```

Comment:
this is so difficult to read, but it's actually `Repox Artifactory (proxy)` 
(hint for the next reviewer)

Reply:
What about renaming the key just to `Repox` to get rid of this problem?

Reply:
This is done already in the first of the three last commits https://github.com/SonarSource/sonar-dotnet/pull/8514/commits/9543920a1328e5988fa6294b75be1ca5bd6ced35.